SU2_CFD/src/iteration/CFluidIteration.cpp

```diff
@@ -318,8 +313,8 @@ void CFluidIteration::SetWind_GustField(CConfig* config, CGeometry** geometry, C
   unsigned long iPoint;
   unsigned short iMGlevel, nMGlevel = config->GetnMGLevels();
 
-  su2double x, y, x_gust, dgust_dx, dgust_dy, dgust_dz, dgust_dt;
-  su2double *Gust, *GridVel, *NewGridVel, *GustDer;
+  su2double x, y, x_gust;
+  su2double *Gust, *GridVel, *NewGridVel;
```

Comment:
This is still not best practice but github doesn't let me comments on the lines I want to change xD
```suggestion
  su2double x, y, x_gust, Gust[3] = {0.0}, NewGridVel[3] = {0.0};
  const su2double* GridVel = nullptr;
```